src/app/api/routes/users.py

```diff
@@ -99,4 +99,5 @@ async def delete_user(user_id: int = Path(..., gt=0), _=Security(get_current_use
     """
     Based on a user_id, deletes the specified user
     """
+    # TODO: Doesn't work when there is a owned device
```

Comment:
I get that this is temporary but I don't understand the comment?

Reply:
Will rephrase but it will yield an exception if the deleted user is the owner_id field of one device.